processing/src/main/java/io/druid/query/aggregation/LongMaxAggregatorFactory.java

```diff
@@ -104,7 +110,7 @@ public Object combine(Object lhs, Object rhs)
   @Override
   public AggregateCombiner makeAggregateCombiner()
   {
-    return new LongAggregateCombiner()
+    return NullHandlingHelper.getNullableCombiner(new LongAggregateCombiner()
```

Comment:
Please extract combiner as a variable and add `return NullHandlingHelper.getNullableCombiner(combiner)` line below